apps/src/lib/util/urlHelpers.js

```diff
@@ -16,6 +17,17 @@ export function pegasus(relativeUrl) {
   return relativeUrl;
 }
 
+/**
+ * Sets the global URL prefix for pegasus
+ * (e.g. "https://code.org") allowing the pegasus()
+ * method above to generate absolute URLs.
+ * @param {string} origin
+ */
+export function setPegasusOrigin(origin) {
```

Comment:
Note: I'm using "origin" here after the [`Location.origin`](https://developer.mozilla.org/en-US/docs/Web/API/Location/origin) property which is the closest to what we actually mean: A scheme (http/https), a hostname (studio.code.org) and, if needed, a port number.